components/View/index.jsx

```diff
@@ -86,6 +95,37 @@ class View extends Component {
   }
 
   /**
+   * Creates an internal reference to an element.
+   * @param {Object} ref The reference to an element.
+   */
+  setRef = (ref) => {
+    this.element = ref;
+  }
+
+  /**
+   * Handles the scroll event of this component's element.
+   */
+  handleScroll = throttle(() => {
+    if (!this.element) {
+      return;
+    }
+
+    const isViewTop = this.element.scrollTop === 0;
+
+    if (isViewTop !== this.props.viewTop) {
+      this.props.setTop(isViewTop);
+    }
+  }, 5);
```

Comment:
Please remove this magic number and use some constant instead.
Also, 5ms seems quite low to me. Can't this be a little higher? Maybe something around 50ms?

Reply:
Done.